### PR TITLE
Add feature for full season statcast batter data

### PR DESCRIPTION
**Pull request by TylerLiu42**
Original PR is here: https://github.com/jldbc/pybaseball/pull/70

### CODE REVIEW

**Comment by schorrm** [comment]
Really good stuff. Two questions:
1. Can you add in the parallel documentation for the function call to docs/statcast_batter.md?
2. Can you tweak the name a bit to fit the typical snake case?
Thank you!

**Comment by schorrm** [comment]
And one note -- this uses f-strings for the first time, which would break backwards compatibility. However, I'm in favor of it, f-strings make much more readable code, and it's 2020. Py3.6+ is not an unreasonable standard.

File: pybaseball/statcast_batter.py [hunk 1]
```diff
@@ -16,3 +16,9 @@ def statcast_batter(start_dt=None, end_dt=None, player_id=None):
         url = 'https://baseballsavant.mlb.com/statcast_search/csv?all=true&hfPT=&hfAB=&hfBBT=&hfPR=&hfZ=&stadium=&hfBBL=&hfNewZones=&hfGT=R%7CPO%7CS%7C=&hfSea=&hfSit=&player_type=batter&hfOuts=&opponent=&pitcher_throws=&batter_stands=&hfSA=&game_date_gt={}&game_date_lt={}&batters_lookup%5B%5D={}&team=&position=&hfRO=&home_road=&hfFlag=&metric_1=&hfInn=&min_pitches=0&min_results=0&group_by=name&sort_col=pitches&player_event_sort=h_launch_speed&sort_order=desc&min_abs=0&type=details&'
         df = split_request(start_dt, end_dt, player_id, url)
         return df
+    
+def statcast_batter_exitVeloBarrels(year, minBBE="q"):
```
Comment (schorrm) [inline pybaseball/statcast_batter.py]: Maybe `statcast_batter_exitvelo_barrels`?

Comment (TylerLiu42) [reply pybaseball/statcast_batter.py]: Fixed and updated doc

**Comment by TylerLiu42** [comment]
> And one note -- this uses f-strings for the first time, which would break backwards compatibility. However, I'm in favor of it, f-strings make much more readable code, and it's 2020. Py3.6+ is not an unreasonable standard.

I noticed there are a few places where string interpolation is used inside a URL. Refactoring such cases to use f-strings could be a worthwhile, if not tedious exercise. Probably won't happen though, if we're being realistic.

**Comment by schorrm** [comment]
Yeah, I think those could be moved over gradually, to make code more concise and clear. I just wanted to note that we're breaking backwards compatibility (if we switch to f-strings, something has to be the first). This is as good as any.

**Comment by schorrm** [comment]
I'm going to merge, I'll update the `__init__.py`

**Pull request merged**